pkg/execution/history/lifecycle.go

```diff
@@ -194,23 +198,26 @@ func (l lifecycle) OnFunctionCancelled(
 	ctx context.Context,
 	id state.Identifier,
 	req execution.CancelRequest,
+	s state.State,
 ) {
+	completedStepCount := int64(len(s.Actions()) + len(s.Errors()))
```

Comment:
**thought:** It could be cool eventually to also add a billable step column as well that follows the calculation method for that. That could be useful if/when we want to show the value to users or use it for our own BI.